### ISSUE TITLE
Any chance of adding an option to zoom in to the game screen?

### ISSUE THREAD

**Issue by spoonmanx** [spiritualized1997/openFPGA-Super-GB#3]
Or a way to not display the borders. I think it would be nicer to see a bit of the border, but to zoom in to fill the screen with mostly just the game screen. Thank You! -Matt

**Comment by TheJCBand** [comment]
I would love a setting to scale the Gameboy image to be exactly the same as using the Pocket's own firmware, just still use the SGB colors

**Comment by syltendo** [comment]
I would love that option too.

**Comment by Rot-gut** [comment]
Agreed, I am really impressed by this core but it would be perfect if it could have zoomed-in option with 8:7 aspect ratio correction.

**Comment by h3llb3nt** [comment]
would also like a zoom feature
